src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The Circle class should not extend Figure directly if Figure is an abstract class intended to set behavior. Consider using an interface to define behaviors that can be shared across different figures.